### PR TITLE
CASSANDRA-17219: December 2021 blog post "Using Arithmetic Operators in Cassandra 4.0"

### PR DESCRIPTION
**Pull request by nonstopdtop**
* Added blog post "Using Arithmetic Operators in Cassandra 4.0"
* Updated blog index with new blog card
* Image added to images/blog

### CODE REVIEW

**Review by michaelsembwever — APPROVED** [review]

File: site-content/source/modules/ROOT/pages/blog/Using-Arithmetic-Operators-in-Cassandra-4.0.adoc [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+:keywords: 
+
+image:blog/using-arithmetic-operators-in-cassandra-4.0-unsplash-bekky-bekks.jpg[operations]
+
+Support for arithmetic operators was added to CQL in Cassandra 4.0 and this blog post describes the important things to know about CQL arithmetics operators, and how we addressed the non-trivial challenges around return types and types inference.
+
+=== Operations on Numbers
+Cassandra supports five arithmetic operators on numbers:
+// adding a table and assuming symbols aren’t an issue. Fingers crossed.
+.Arithmetic Operators in 4.0
```
Comment (ErickRamirezAU) [inline site-content/source/modules/ROOT/pages/blog/Using-Arithmetic-Operators-in-Cassandra-4.0.adoc:17]: The table title doesn't render correctly. I have a feeling there needs to be an empty line before the title:

```suggestion

.Arithmetic Operators in 4.0
```

File: site-content/source/modules/ROOT/pages/blog/Using-Arithmetic-Operators-in-Cassandra-4.0.adoc [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+table_name,
+Task_id,
+(progress * 100) / total AS percentage_of_completion
+FROM system_views.sstable_tasks
+WHERE kind = 'compaction'
+ALLOW FILTERING;
+----
+=== The Important Role of Return Types
+An important thing to take into account when using operators is the return type of the operation. In Cassandra the return type is based on the type of the operands and can be determined using the following table:
+.Return Type Based on Operand in Apache Cassandra
```
Comment (ErickRamirezAU) [inline site-content/source/modules/ROOT/pages/blog/Using-Arithmetic-Operators-in-Cassandra-4.0.adoc:41]: Similar to line 16 above, the table title doesn't render and I think requires a blank line before it:

```suggestion

.Return Type Based on Operand in Apache Cassandra
```

File: site-content/source/modules/ROOT/pages/blog/Using-Arithmetic-Operators-in-Cassandra-4.0.adoc [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+[cols="1,1"]
+|===
+|Operator |Description
+|+ |addition 
+|- |negates operand/substraction
+|* |multiplication
+|/ |division
+|% |returns the remainder of a division
+|===
+Those operators can be used in the `SELECT` `INSERT`, `UPDATE,` and `DELETE` statements to perform some computations. For example, if you wish to know the percentage of completion of the currently running compactions, you can query the *sstable_tasks* system view using the following query:
```
Comment (ErickRamirezAU) [inline site-content/source/modules/ROOT/pages/blog/Using-Arithmetic-Operators-in-Cassandra-4.0.adoc]:
```suggestion
Those operators can be used in the `SELECT` `INSERT`, `UPDATE,` and `DELETE` statements to perform some computations. For example, if you wish to know the percentage of completion of the currently running compactions, you can query the `sstable_tasks` system view using the following query:
```

**Review by ErickRamirezAU — CHANGES_REQUESTED** [review]
I've confirmed that a blank line is needed for the table titles to work. I'll add screenshots in the JIRA. Cheers! 🍻

**Review by ErickRamirezDS — APPROVED** [review]

**Review by ErickRamirezAU — APPROVED** [review]

**Comment by michaelsembwever** [comment]
https://cassandra.staged.apache.org/ updated again.

**Comment by michaelsembwever** [comment]
manually merged with 0676488d6169ff206eba074374f42822618dabf7
deployed live with 29aec85222c7c3623be699c81d7168d948e2a456

**Pull request closed without merging**